### PR TITLE
Fix/documentation

### PR DESCRIPTION
**Pull request by dcuestam**
Make sure you have checked _all_ steps below.

### Description
- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [ ] My commits have been squashed if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

## License
- [ ] Add a [Mozilla Public License 2.0 license header](http://mozilla.org/MPL/2.0/) to all the new files.

### Documentation
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.

### CODE REVIEW

File: tests/unit_tests/array_unit_tests.py [hunk 1]
```diff
@@ -294,6 +294,11 @@ def testIadd(self):
         a += b
         np.testing.assert_array_equal(a.to_numpy(), np.array([2, 4, 6, 8]))
 
+    def testIaddSelfArray(self):
```
Comment (avilchess) [inline tests/unit_tests/array_unit_tests.py]: David, delete this test, as it may collision with PR 37.

File: khiva/matrix.py [hunk 2]
```diff
@@ -99,6 +103,11 @@ def stomp_self_join(time_series, subsequence_length):
     """ Stomp algorithm to calculate the matrix profile between `t` and itself using a subsequence length of `m`.
     This method filters the trivial matches.
 
+
```
Comment (avilchess) [inline khiva/matrix.py]: There is an extra empty line.

File: khiva/matrix.py [hunk 3]
```diff
@@ -77,6 +77,10 @@ def find_best_n_motifs(profile, index, m, n, self_join=False):
 def stomp(first_time_series, second_time_series, subsequence_length):
     """ Stomp algorithm to calculate the matrix profile between `ta` and `tb` using a subsequence length of `m`.
```
Comment (avilchess) [inline khiva/matrix.py]: Does this method filter trivial matches? if so, please add, "This method filters the trivial matches.", as the one below

File: tests/unit_tests/array_unit_tests.py (hunk 1 above)
Comment (dcuestam) [reply tests/unit_tests/array_unit_tests.py]: Don't worry about that, I will pull the changes and fix it when PR 37 is merged. Right now this branch have a pull from the branch of PR 37.

File: khiva/matrix.py (hunk 3 above)
Comment (dcuestam) [reply khiva/matrix.py]: This method doesn't filter trivial matches as it consider that both time series are different.

File: khiva/matrix.py (hunk 2 above)
Comment (dcuestam) [reply khiva/matrix.py]: Nice!

**Pull request merged**